CMakeLists.txt

```diff
@@ -1159,10 +1159,22 @@ if(IREE_BUILD_COMPILER)
     ADD_TO iree-install-tools
   )
 
+  iree_add_install_target(
+    NAME iree-install-tools-compiler-extra
+    COMPONENT IREETools-CompilerExtra
+    ADD_TO iree-install-tools
```

Comment:
Add a todo here to not ADD_TO like this but to include this in the tarball build that needs it versus having it along for the ride and inseparable.